axonserver/src/main/java/io/axoniq/axonserver/localstorage/LocalEventStore.java

```diff
@@ -183,10 +184,11 @@ public void onCompleted() {
             }
 
             private Void error(Throwable exception) {
+                exception = ConcurrencyExceptions.unwrap(exception);
                 if( isClientException(exception)) {
-                    logger.warn("Error while storing events: {}", exception.getMessage());
+                    logger.info("{}: Error while storing events: {}", context, exception.getMessage());
                 } else {
-                    logger.warn("Error while storing events", exception);
+                    logger.warn("{}: Error while storing events", context, exception);
```

Comment:
For me this feels like a real `Error`, not a `Warning`, since an error in storing events is something that we really want to avoid.
Or maybe I lack some context and I'm too pessimistic.

Reply:
Some of the server exceptions may be real errors, but not all of them. We should probably do some more cleaning up of the exception handling to determine the correct logging levels.
Not changing this at this moment.